lib/matplotlib/container.py

```diff
@@ -18,7 +18,7 @@ def __repr__(self):
     def __new__(cls, *kl, **kwargs):
         return tuple.__new__(cls, kl[0])
 
-    def __init__(self, kl, label=None):
+    def __init__(self, kl, label=None, lorder=''):
```

Comment:
I think you should use `None` instead of `''` -- not 100% positive on that, though.